pkg/tfvars/openstack/openstack.go

```diff
@@ -179,19 +174,46 @@ func TFVars(
 		additionalNetworkIDs = mastermpool.AdditionalNetworkIDs
 	}
 
-	var additionalSecurityGroupIDs []string
-	if mastermpool != nil {
-		additionalSecurityGroupIDs = mastermpool.AdditionalSecurityGroupIDs
+	// defaultMachinesPort carries the machinesSubnet (and its resolved
+	// network) if provided.
+	var defaultMachinesPort *terraformPort
```

Comment:
I see we're substituting MachinesSubnet here, but what about the case where it isn't defined and we're using a terraform-created network?

Assuming there's some terraform logic to substitute that later, couldn't we just put it all here? The split logic is harder to follow.

Reply:
We can only leave blanks where the Terraform-created network should be: we don't have that data. I will try to make it more explicit that `nil` means that.